Task2/aga/TicTacToe/TicTacToe/TheGame.xaml.cs

```diff
@@ -0,0 +1,242 @@
+﻿using System;
```

Comment:
A lot is happening in this one file. Code-behind is part of View, so try to only keep logic that is required by the View here. Business logic (in our case it would be mostly game mechanics) should be moved to Model and all logic required to display Model on the View should happen in ViewModel